test/support/single_cov.rb

```diff
@@ -0,0 +1,67 @@
+class SingleCov
+  COVERAGES = []
+
+  class << self
+    def expect(file, percent)
+      COVERAGES << [file, percent]
+    end
+
+    def verify!(result)
+      COVERAGES.each do |file, expected_percent|
+        uncovered!(file) unless coverage = result[File.expand_path(file)]
+
+        line_count = coverage.count { |l| l }
```

Comment:
I guess we don't need to pass the block here? Or are some of these elements `false`?